commands/playground/youtubeQueue.js

```diff
@@ -0,0 +1,30 @@
+const ytdl = require("ytdl-core");
```

Comment:
why is the q in playground?

Reply:
Because I didnt know how git branching worked lmao